packages/select/src/common/listItemsProps.ts

```diff
@@ -98,6 +98,14 @@ export interface IListItemsProps<T> extends IProps {
      */
     onQueryChange?: (query: string, event?: React.ChangeEvent<HTMLInputElement>) => void;
 
+    /**
+     * Whether the active selection should be reset when a query is entered.
+     * If you search and your current active item falls out of scope,
+     * the first matching item will become the active selection.
```

Comment:
@switz this second sentence is not accurate... changing the query _always_ resets to first active item. did you intend it to only reset if the active item falls out of scope?

Reply:
That's a great point -- I'm not sure which is better to be honest. I can update the docs in the meantime.